waku/v2/protocol/waku_store/waku_store.nim

```diff
@@ -245,7 +251,7 @@ proc findIndex*(msgList: seq[IndexedWakuMessage], index: Index): Option[int] =
       return some(i)
   return none(int)
 
-proc paginateWithIndex*(list: seq[IndexedWakuMessage], pinfo: PagingInfo): (seq[IndexedWakuMessage], PagingInfo) =
+proc paginate*(list: seq[IndexedWakuMessage], pinfo: PagingInfo): (seq[IndexedWakuMessage], PagingInfo, HistoryResponseError) =
```

Comment:
Perhaps if this defaults to `OK` I imagine you wouldn't have to explicitly set `HistoryResponseError.OK` in so many places?

Reply:
The default is OK, however, as part of the unit testing, I need to check that the different execution paths of `paginate` that yield OK behave as expected and do not produce an error.
Please let me know if there is a specific part of the code that the use of OK is unnecessary and can be avoided.

Reply:
BTW, as I am using `return` to terminate the proc, I cannot leave the third output value unspecified.